README.md

```diff
@@ -42,7 +42,7 @@ If you have followed the instructions for [installing Kubeapps](./docs/user/gett
 - [Detailed installation instructions](chart/kubeapps/README.md)
 - [Kubeapps Dashboard documentation](./docs/user/dashboard.md)
 - [Kubeapps components](./docs/architecture/overview.md)
-- [Roadmap](https://github.com/kubeapps/kubeapps/wiki/Roadmap)
+- [Roadmap](https://github.com/kubeapps/kubeapps/milestones)
```

Comment:
Might also be worth including a link to the project board? https://github.com/kubeapps/kubeapps/projects/11